models/Recording.js

```diff
@@ -317,6 +317,18 @@ module.exports = function(sequelize, DataTypes) {
     }
   };
 
+  /* eslint-disable indent */
```

Comment:
why disable this here?

Reply:
Just disables the indenting lint messages.  I've indented it so that you can see which requirement is on which table.